concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-227	227	abatir	download			Verb	3.77450980392157	6	5	
+Desrochers-2010-330-228	228	abrir	open			Verb	6.55882352941176	5	275	6.45
+Desrochers-2010-330-229	229	acudir	go to			Verb	4.82352941176471	6	141	4.91
+Desrochers-2010-330-230	230	afilar	sharpen	SHARPEN (SOMETHING)	251	Verb	4.63725490196079	6	6	
+Desrochers-2010-330-231	231	aguar	water down			Verb	3.73529411764706	5		
+Desrochers-2010-330-232	232	alumbrar	to illuminate			Verb	4.56862745098039	8	16	4.77
+Desrochers-2010-330-233	233	apaciguar	appease			Verb	3.5	9	6	
+Desrochers-2010-330-234	234	apagar	turn off			Verb	6.5	6	39	5.97
+Desrochers-2010-330-235	235	asir	summarize			Verb	2.43572184416938	4	6	
+Desrochers-2010-330-236	236	atrapar	trap			Verb	4.95098039215686	7	55	5.29
+Desrochers-2010-330-237	237	avisar	notify			Verb	6.02941176470588	6	45	5.81
+Desrochers-2010-330-238	238	balbucear	babble			Verb	3.46078431372549	9	5	
+Desrochers-2010-330-239	239	besar	kiss	KISS	578	Verb	6.52941176470588	5	34	6.85
+Desrochers-2010-330-240	240	brotar	sprout	SPROUT (VERB)	2076	Verb	4.89	6	26	4.13
+Desrochers-2010-330-241	241	bullir	boil	BOIL	2493	Verb	2.41176470588235	6	1	
+Desrochers-2010-330-242	242	buscar	search	LOOK FOR	1468	Verb	6.29411764705882	6	457	6.39
+Desrochers-2010-330-243	243	calar	stall			Verb	4.2843137254902	5	3	
+Desrochers-2010-330-244	244	calcificar	calcify			Verb	3.08823529411765	10		
+Desrochers-2010-330-245	245	calentar	heat	HEAT UP	1341	Verb	6.41176470588235	8	31	6.74
+Desrochers-2010-330-246	246	cantar	sing	SING	1261	Verb	6.34313725490196	6	137	6.21
+Desrochers-2010-330-247	247	cascar	click			Verb	4.65686274509804	6	2	
+Desrochers-2010-330-248	248	catar	taste	TASTE (SOMETHING)	21	Verb	4.34313725490196	5	3	
+Desrochers-2010-330-249	249	cazar	hunting	HUNT	1435	Verb	5.30392156862745	5	43	5.2
+Desrochers-2010-330-250	250	cerrar	close	SHUT	1046	Verb	6.64705882352941	6	197	6.46
+Desrochers-2010-330-251	251	cobijar	shelter			Verb	3.2156862745098	7	6	
+Desrochers-2010-330-252	252	colegir	to deduce			Verb	1.36274509803922	7		
+Desrochers-2010-330-253	253	contar	count	COUNT	1420	Verb	6.31372549019608	6	364	6.52
+Desrochers-2010-330-254	254	cortar	cut	CUT	1432	Verb	6.29411764705882	6	101	5.73
+Desrochers-2010-330-255	255	cubrir	cover	COVER	761	Verb	5.5	6	103	6.11
+Desrochers-2010-330-256	256	cumplir	meet	MEET	1831	Verb	5.86274509803922	7	285	6.04
+Desrochers-2010-330-257	257	declamar	declaim			Verb	2.2156862745098	8	3	
+Desrochers-2010-330-258	258	delinquir	offending			Verb	3.06862745098039	9	2	
+Desrochers-2010-330-259	259	embargar	seize	SEIZE	855	Verb	4.04901960784314	8	5	
+Desrochers-2010-330-260	260	embeber	embed			Verb	2.17647058823529	7	1	
+Desrochers-2010-330-261	261	embolar	embolish			Verb	1.98039215686275	7		
+Desrochers-2010-330-262	262	entablar	to set up			Verb	3.94117647058824	8	15	
+Desrochers-2010-330-263	263	erguir	erect			Verb	2.93137254901961	6	8	
+Desrochers-2010-330-264	264	escurrir	draining			Verb	5.26470588235294	8	2	
```

Comment:
"ablaufen" (von flüssigkeit), not clear if "draining" is a good translation for the verb.

Reply:
Yes, "to drain" or something like that. It could be intended as an antonym to "embeber" (even though the closest antonym is "mojar").